source/js/nav_jquery.js

```diff
@@ -1,8 +1,54 @@
 // Executes functions as soon as the page is fully loaded.
-$(document).ready(function() {
+$(document).ready(function () {
+    var isDesktop = DesktopCheck();
```

Comment:
any particular reason why these are `var` while most later variables are `let`?

Reply:
`var`s have been changed to `let`s